go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/agnivade/levenshtein v1.1.1
 	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
 	github.com/bytecodealliance/wasmtime-go/v3 v3.0.2
-	github.com/containerd/containerd v1.7.11
+	github.com/containerd/containerd v1.7.0-beta.2
```

Comment:
This does not look like an official release. So probably better to hold off on this change.